libs/community/tests/unit_tests/test_vertex_ai_search.py

```diff
@@ -313,3 +314,67 @@ def test_convert_unstructured_search_response_extractive_answers(
         assert "relevance_score" not in documents[1].metadata
         assert "previous_segments" not in documents[1].metadata
         assert "next_segments" not in documents[1].metadata
+
+
+def test_custom_embedding_with_valid_values() -> None:
+    """
+    Test with a valid custom embedding model and field path.
+    """
+    embeddings = FakeEmbeddings(size=100)
+    retriever = VertexAISearchRetriever(
```

Comment:
please, take a look at the example above. You need to mock the SearchServiceClient, otherwise unit tests fail because of no auth on the github worker.

Reply:
Thanks, ok, I tried again. I hope it works.